Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -1512,6 +1513,12 @@ bool RenderBox::nodeAtPoint(const HitTestRequest& request, HitTestResult& result
 {
     LayoutPoint adjustedLocation = accumulatedOffset + location();
 
+    // Exit early if no children can be hit.
+    LayoutRect overflowRect = visualOverflowRect();
+    overflowRect.moveBy(adjustedLocation);
+    if (!locationInContainer.intersects(overflowRect))
+        return false;
+
```

Comment:
I see you closed this one! sounds good to me. I looked at it a bit and not sure if I could justify the potential regression.

Reply:
It is silly error on my end that I closed the branch while trying to clean-up old ones. My bad! It does not have any test failures, you mean any future potential regression due to this since it lacks testcase?

Reply:
hah, ok. well, this looks like a small perf win vs. potential functional regression (when visual overflow is incorrectly computed). I suggest you seek a second opinion on this. Simon maybe?

Reply:
Sure. I will ask Simon, if he give his opinion and will DM him. :-)

Reply:
In WebKit, `visualOverflowRect()` can be affected by compositing behavior so it's not easy for me to say if this is correct.

Reply:
Hmm.. @alanbaradlay & @smfr - I think we can park it just to be safe to avoid any future regressions. I am closing this PR but thanks for your input. Thanks!